README.md

```diff
@@ -37,7 +37,7 @@ shortcut by default.
 
 `mate-hud.py` reads one gsettings keys:
 
-  * `org.mate.hud`: `shortcut` (Default: <Super>Alt_L)
+  * `org.mate.hud`: `shortcut` (Default: `"<Super>Alt_L"`)
```

Comment:
I would use single quotes instead of double quotes, as that's what gsettings/dconf expect:
```
$ gsettings get org.mate.hud shortcut
'Alt_L'
```
Also, note that the actual default in code is `'Alt_L'`, not `'<Super>Alt_L'`, so it'd be good to update that.